client.go

```diff
@@ -103,6 +106,8 @@ func (c *Client) newRequest(method, requestPath string, query url.Values, body i
 
 	if c.config.APIKey != "" {
 		req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", c.config.APIKey))
+	} else if c.config.OrgID != 0 {
```

Comment:
I know this is already merged, but this line is wrong. This two cases are not mutually exclusive.

Reply:
You can't set an X-Grafana-Org-Id header when authenticating with an APIKey (well, you can set it, but it will be ignored).

APIKeys are already scoped to a specific org within Grafana.

Reply:
@woodsaj Thanks, was about to write the same :)

Reply:
Sorry for the noise, at first glimpse it seemed wrong.